install/install.go

```diff
@@ -266,11 +266,16 @@ func Reinstall(ctx context.Context, ps client.PackageState, state client.GooGetS
 	pi := goolib.PackageInfo{Name: ps.PackageSpec.Name, Arch: ps.PackageSpec.Arch, Ver: ps.PackageSpec.Version}
 	logger.Infof("Starting reinstall of %s.%s, version %s", pi.Name, pi.Arch, pi.Ver)
 	fmt.Printf("Reinstalling %s.%s %s and dependencies...\n", pi.Name, pi.Arch, pi.Ver)
+
 	// Fix for package install by older versions of GooGet.
-	if ps.LocalPath == "" {
+	if ps.LocalPath == "" && ps.UnpackDir != "" {
 		ps.LocalPath = ps.UnpackDir + ".goo"
 	}
 
+	if ps.LocalPath == "" {
```

Comment:
It would affect any reinstall since 1. we don't know where the package is locally and 2. we can't redownload it without knowing where to redownload it to.